data/json/professions.json

```diff
@@ -5158,5 +5158,81 @@
                 "panties"
             ]
         }
+    },
+        {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Mad Bionics Scientist",
+        "description": "Before all this you studied bionics. You tried your experiments on animals and yourself.", 
+        "traits": [
+            "ANIMALDISCORD",
+            "PSYCHOPATH"
+        ],
+        "skills": [
+            {"level": 4, "name": "electronics"},
+            {"level": 3, "name": "mechanics"}
+        ],
+        "points": 4,
+        "CBMs": [
+            "bio_alarm",
+            "bio_geiger",
+            "bio_armor_head",
+            "bio_batteries"
```

Comment:
Those are incredibly random and arbitrary bionics without a common theme.

Reply:
>You tried your experiments on yourself...
 
I thought it'd be fitting to have random bionics.